app/views/bookmarks/_bookmark_blurb_short.html.erb

```diff
@@ -1,5 +1,6 @@
 <% # This partial requires local variable 'bookmark' %>
-<li class="user<% if is_author_of?(bookmark) %> own<% end %> short blurb group" role="article">
+<% # user own short blurb group bookmarker-id %>
+<li class="user<% if is_author_of?(bookmark) %> own<% end %> short blurb group <%= bookmarker_id_for_css_classes(bookmark) %>" role="article">
```

Comment:
This line is long enough that I think it might make sense to use the helper method, though I think you could make an argument the other way as well.